homeassistant/components/application_credentials/__init__.py

```diff
@@ -227,9 +256,11 @@ async def _get_platform(
             err,
         )
         return None
-    if not hasattr(platform, "async_get_authorization_server"):
+    if not hasattr(platform, "async_get_authorization_server") and not hasattr(
+        platform, "async_get_auth_implementation"
+    ):
         raise ValueError(
-            f"Integration '{integration_domain}' platform application_credentials did not implement 'async_get_authorization_server'"
+            f"Integration '{integration_domain}' platform application_credentials did not implement 'async_get_authorization_server' or 'async_get_auth_implementation'"
```

Comment:
Please break long strings around 88 characters per line.

Reply:
I am not yet deprecating other config options in the schema as part of the move to application credentials. Is that an issue? Those need more config or options flow support or their own deprecation plan.

Reply:
👍🏼